tests/psp34/psp34.tests.ts

```diff
@@ -1,14 +1,16 @@
 import { bnArg, expect, setupContract, fromSigner } from '../helpers'
 
-describe('MY_PSP721', () => {
+describe('MY_PSP34', () => {
     async function setup() {
-        return setupContract('my_psp721', 'new')
+        return setupContract('my_psp34', 'new')
     }
 
     async function setup_receiver() {
-        return setupContract('psp721_receiver', 'new')
+        return setupContract('psp34_receiver', 'new')
     }
 
+    // TODO: add tests for total_supply and collection_id
```

Comment:
Let's add test for both, in case of `collection_id` check that transaction is reverted

Reply:
working on it